scripts/apps/authoring/authoring/services/AuthoringService.ts

```diff
@@ -497,7 +497,8 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
             helpers.filterDefaultValues(diff, origItem);
 
             if (_.size(diff) > 0) {
-                return api.save('archive', origItem, diff).then((__item) => {
+                return api.save('archive', origItem, diff, {},
+                    {publish_from_personal: appConfig?.features?.publishFromPersonal}).then((__item) => {
```

Comment:
We should avoid having undefined keys in object. Either add a default using `appConfig?.features?.publishFromPersonal ?? false` or only add the key if it's true.

```
if (appConfig?.features?.publishFromPersonal === true) {
    params['publish_from_personal'] = true;
}
```